packages/angular/projects/tx-native-angular-sdk/tests/T.decorator.spec.ts

```diff
@@ -1,77 +1,55 @@
-  };
-
   @Component({
-    selector: 'test-cmp',
-    template: '<div>{{testProperty}}</div>',
-    styles: [],
+    template: '<div>{{ testProperty }}</div>',
   })
-  class TestComponent implements OnInit {
-    @T('not-trans-dec', { _key: 'test' })
-      testProperty: any;
-
-    ngOnInit() {
-    }
+  class TestComponent {
+    @T('not-trans-dec', { _key: 'test' }) testProperty!: string;
   }
 
   @Component({
-    selector: 'test-cmp',
-    template: '<div>{{testProperty}}</div>',
-    styles: [],
+    template: '<div>{{ testProperty }}</div>',
   })
-  class TestWithInstanceComponent implements OnInit {
-    @T('not-trans-dec', { _key: 'test' }, instanceConfig)
-      testProperty: any;
-
-    ngOnInit() {
-    }
+  class TestWithInstanceComponent {
+    @T('not-trans-dec', { _key: 'test' }, 'alias') testProperty!: string;
   }
 
-  let component: TestComponent;
-  let fixture: ComponentFixture<TestComponent>;
-
-  let componentWithInstance: TestWithInstanceComponent;
-  let fixtureWithInstance: ComponentFixture<TestWithInstanceComponent>;
-
   beforeEach(() => {
-    spyOn(tx, 'translate').and.returnValue('ok-translated-dec');
+    TestBed.configureTestingModule({ teardown: { destroyAfterEach: false } });
   });
 
   beforeEach(() => {
-    fixture = TestBed.createComponent(TestComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
-
-    fixtureWithInstance = TestBed.createComponent(TestWithInstanceComponent);
-    componentWithInstance = fixtureWithInstance.componentInstance;
-    fixtureWithInstance.detectChanges();
+    spyOn(tx, 'translate').and.returnValue('ok-translated-dec');
   });
 
   it('should test the decorator T', () => {
-    const compiled = fixture.debugElement.nativeElement;
-    expect(tx.translate).toHaveBeenCalled();
-    expect((compiled as HTMLDivElement).innerHTML)
-      .toContain('ok-translated-dec');
+    const fixture = TestBed.createComponent(TestComponent);
+    const component = fixture.componentInstance;
+
+    fixture.detectChanges();
+    expect(tx.translate).toHaveBeenCalledTimes(2); // Why twice?
+    expect(tx.translate).toHaveBeenCalledWith('not-trans-dec', { _key: 'test' });
+    expect(component.testProperty).toBe('ok-translated-dec');
+
+    const compiled: HTMLElement = fixture.debugElement.nativeElement;
+    expect(compiled.innerHTML).toContain('ok-translated-dec');
   });
 
   it('should test the decorator T with an instance', () => {
-    const compiled = fixtureWithInstance.debugElement.nativeElement;
-    expect(tx.translate).toHaveBeenCalled();
-    setTimeout(function () {
-      expect((compiled as HTMLDivElement).innerHTML)
-        .toContain('ok-translated-dec');
-    }, 1000);
+    const fixtureWithInstance = TestBed.createComponent(TestWithInstanceComponent);
+    const component = fixtureWithInstance.componentInstance;
+
+    fixtureWithInstance.detectChanges();
+    expect(tx.translate).toHaveBeenCalledTimes(2); // Why twice?
```

Comment:
I think we can ignore this, and use a simple `hasBeenCalled()` . I couldn't find any reason why it's called twice... probably some internal mechanism for decorators? Let's assume that is working fine.